ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.js

```diff
@@ -24,6 +25,8 @@ import EditGasToolTip from '../edit-gas-tooltip/edit-gas-tooltip';
 import InfoTooltip from '../../../ui/info-tooltip';
 import { useCustomTimeEstimate } from './useCustomTimeEstimate';
 
+const HeartBeat = () => (process.env.IN_TEST ? null : <LoadingHeartBeat />);
```

Comment:
In the `LoadingHeartBeat` component itself, let's just do the `process.env.IN_TEST` check there and return null, so that we don't have to do conditional checks like this

Reply:
good point, I just fixed